### DISCUSSION TITLE
Equation editing?

### DISCUSSION THREAD

**Discussion by spagdoon0411** [mdx-editor/editor#35] [General]
Hey, y'all! I was wondering if anyone's found a way to allow users to embed/enter LaTeX block or inline equations. Are there any plans to add equation-editing as a feature?

**Comment by petyosi** [comment]
@spagdoon0411 - do you have any specific React components that can be used for the purpose? Or would you rather have the LaTeX edited as a plain text/code?

**Reply by noteverseio** [reply]
mathlive can be a good option to edit equations visually.

**Reply by spagdoon0411** [reply]
I think I'm a fan of MathLive--thanks!
